src/main/java/storagesystem/model/Team.java

```diff
@@ -0,0 +1,80 @@
+package storagesystem.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A team has a list of users that belong to the same team.
+ * A team has an inventory of items which it can browse.
+ * A user may borrow items from other teams inventory for its own team .
+ */
+public class Team {
+    private String name;
+    private final List<Item> inventory = new ArrayList<>(); // maybe itemIDs instead?
```

Comment:
Perhaps, but should be consistent with Users and rest of application which currently are ID's